dev/linux/bashrc.sh

```diff
@@ -10,10 +10,39 @@ restore_e() {
 }
 trap restore_e EXIT
 
+function recompile-mamba () {
+  if [ -d "/opt/mamba-src" ]; then
+    pushd /opt/mamba-src/
+    sudo /opt/conda/bin/cmake -B build/ \
+        -DBUILD_LIBMAMBA=ON \
+        -DBUILD_SHARED=ON \
+        -DCMAKE_INSTALL_PREFIX=/opt/conda \
+        -DCMAKE_PREFIX_PATH=/opt/conda \
+        -DBUILD_LIBMAMBAPY=ON
+    sudo /opt/conda/bin/cmake --build build/ -j
+    sudo make install -C build/
+    popd
+  else
+    echo "ERROR: path '/opt/mamba-src' not mounted so no libmamba to recompile"
+  fi
+}
+
 sudo /opt/conda/condabin/conda install -y -p /opt/conda \
     --file /opt/conda-libmamba-solver-src/dev/requirements.txt \
     --file /opt/conda-libmamba-solver-src/tests/requirements.txt
 
+if [ -d "/opt/mamba-src" ]; then
+  # remove "sel(win)" in environment yaml hack since conda does not understand
+  # libmamba specific specs
+  sed '/sel(.*)/d' /opt/mamba-src/mamba/environment-dev.yml > /tmp/mamba-environment-dev.yml
```

Comment:
TIL `sed` (with no flags) understands `()` as literal characters, not grouping ones!

Reply:
Yeah I learned that while doing it too. Apparently `\(` and `\)` are actually the grouping characters... so it is the reverse of what we'd typically expect.